modules/HapticGlove_module/src/MotorEstimation.cpp

```diff
@@ -0,0 +1,100 @@
+/**
+ * @file MotorEstimation.cpp
+ * @authors  Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2020 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2020
+ */
+
+#include <MotorEstimation.hpp>
+#include <iostream>
+using namespace HapticGlove;
+
+Estimator::Estimator(const double& dt, const Eigen::MatrixXd& R, const Eigen::MatrixXd& Q)
+{
+    // hard coded model
+    // reference issue: https://github.com/ami-iit/element_retargeting-from-human/issues/145
```

Comment:
Nitpick, this code is public, but this issue is not public. I think it is ok to keep the link, but maybe you can add some details